manifests/secondarydns.yaml

```diff
@@ -0,0 +1,124 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: secondary
+---
+apiVersion: v1
+data:
+  USER_DOMAIN: secondary.io
+  DOMAIN_HOST_IP: 127.0.0.1
+  Corefile: |
+    .:53 {
+        auto {
+          directory /zones db\.(.*) {1}
+          reload 45s
+        }
+        reload
+        log
+    }
+kind: ConfigMap
+metadata:
+  name: secondary-dns
+  namespace: secondary
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: secondary
+rules:
+- apiGroups:
+  - kubevirt.io
+  resources:
+  - virtualmachineinstances
+  verbs:
+  - get
+  - list
+  - watch
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: secondary
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: secondary
+subjects:
+- kind: ServiceAccount
+  name: default
```

Comment:
Why do you use the default service account and not creating a new specific one?
Doesn't it mean that any pod created in `secondary` namespace will get the permissions to get/list/watch the virtualmachineinstances?

Reply:
yes it does but we arent going to add there more pods, and if someone has permissions to create there pods he is basically strong enough to also use other service account

I think we can play with those things later if needed, anyhow i saw that kmp IIRC (not sure) does the same